src/Libraries/Microsoft.Extensions.Http.Resilience/README.md

```diff
@@ -4,6 +4,11 @@ Resilience mechanisms for `HttpClient` built on the [Polly framework](https://ww
 
 ## Install the package
 
+> [!IMPORTANT]
+> If you're using the `Grpc.Net.ClientFactory` package then update it to the version `>=2.64.0`
+> before installing the `Microsoft.Extensions.Http.Resilience` package. This will help you to
+> avoid issues like the [following](https://github.com/dotnet/extensions/issues/4924).
```

Comment:
Having a second thought now... What do you think of adding a "Known Issues" section before "Feedback & Contributing" section?

And in there say something along the lines of:
````
Ensure `Grpc.Net.ClientFactory` v2.64.0 or later is installed, if you receive the following exception:
```
InvalidOperationException: The ConfigureHttpClient method is not supported when creating gRPC clients.
```
````

I don't know if there's any good to link to #4924. What would a developer gain from that thread?

Reply:
@gewarren would you have any suggestions here? We'll need to add a similar note to learn.ms as well.
Thanks

Reply:
> Having a second thought now... What do you think of adding a "Known Issues" section before "Feedback & Contributing" section?

I think that is better what we currently have. Having "Known Issues" section explaining why users need to update `Grpc.Net.ClientFactory` makes perfect sense.

Reply:
Sure, if it's a _known issue_ - call it out as such. I support that idea!

Reply:
I wonder if we should just add some targets file in the package that checks for this before  build and if this is the case we log a warning. If we think this is an important thing to call out, then doing that will bring this error to devs faces as opposed to keeping it on docs only.

Reply:
@joperezr it's an interesting idea.

Reply:
@iliar-turdushev the target would look something like this:
```xml
  <!-- 
    Check whether the project is referencing Grpc.Net.ClientFactory/2.64.0 or older.
    If the vesion is newer warn the user to update.
    -->
  <Target Name="_CheckGrpcNetClientFactoryVersion"
          AfterTargets="ResolveReferences"
          Condition=" '$(SuppressCheckGrpcNetClientFactoryVersion)' != 'true' ">
    <ItemGroup>
      <_GrpcNetClientFactory Include="@(PackageReference)"
                             Condition=" '%(PackageReference.Identity)' == 'Grpc.Net.ClientFactory' AND $([MSBuild]::VersionLessThan('%(PackageReference.Version)', '2.64.0')) " />
    </ItemGroup>

    <Error Condition="@(_GrpcNetClientFactory->Count()) &gt; 0"
           Text="Grpc.Net.ClientFactory v2.64.0 or later is required (see https://github.com/dotnet/extensions/issues/4924 for more details). The check can be suppressed with SuppressCheckGrpcNetClientFactoryVersion=true." />
  </Target>
```

I performed some light tests on the repro provided in #4924. Please test other scenarios (including a project using [CPM](https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management)).

Reply:
@RussKie Thank you very much for help. I extended your code sample a little bit and added the check. Please, review it and share your feedback. Also I added "Known issues" section describing the issue and how to fix or suppress it.